nerfstudio/viewer/viewer_elements.py

```diff
@@ -60,17 +61,30 @@ class ViewerClick:
     The direction of the click if projected from the camera through the clicked pixel,
     in world coordinates
     """
+    screen_pos: Tuple[float, float]
+    """The screen position of the click in screen coordinates."""
```

Comment:
Maybe specify that this is also normalized [0,1]?

Reply:
Resolved in https://github.com/nerfstudio-project/nerfstudio/pull/3014/commits/d9e6c889ad7efde630ec8497c1818b3168cf6a4e !